source/org/zfin/expression/presentation/FishExpressionSummaryController.java

```diff
@@ -92,36 +92,26 @@ protected String getFishExpressionFigureSummaryStandard(@RequestParam String fis
 
     }
 
-    @RequestMapping("/fish-expression-figure-summary-envgroup")
-    protected String getFishExpressionFigureSummaryEnvGroup(@RequestParam String fishZdbID,
-                                                                @RequestParam String geneZdbID,
-                                                                @RequestParam boolean imagesOnly,
-                                                                @RequestParam String envGroup,
-                                                                Model model) {
-        Fish fish = getMutantRepository().getFish(fishZdbID);
-
-        if (fish == null) {
-            model.addAttribute(LookupStrings.ZDB_ID, fishZdbID);
-            return LookupStrings.RECORD_NOT_FOUND_PAGE;
-        }
-
-        Marker gene = RepositoryFactory.getMarkerRepository().getMarkerByID(geneZdbID);
-
-        if (gene == null) {
-            model.addAttribute(LookupStrings.ZDB_ID, fishZdbID);
-            return LookupStrings.RECORD_NOT_FOUND_PAGE;
-        }
 
-        ExpressionSummaryCriteria expressionCriteria = FigureService.createExpressionCriteriaEnvironmentGroup(fish, gene, imagesOnly, envGroup);
-        model.addAttribute("expressionCriteria", expressionCriteria);
-        List<FigureSummaryDisplay> figureSummaryDisplayList = FigureService.createExpressionFigureSummary(expressionCriteria);
-        model.addAttribute("figureSummaryDisplayList", figureSummaryDisplayList);
-
-        model.addAttribute(LookupStrings.DYNAMIC_TITLE, fish.getName() + " Expression Figure Summary");
-        return "expression/genotype-figure-summary";
-
-    }
 
+    //Not seeing any traffic to this page in our access logs other than some hits from bingbot
```

Comment:
Yes, remove the method if it is not being called from within our app.

Reply:
I removed it